examples/forms.amp.html

```diff
@@ -535,5 +535,18 @@ <h4 id="header1">Using button[type=submit]</h4>
 </form>
 
 
+<h4 id="header1">Redirect After POST (works only if running on localhost)</h4>
+<form method="post"
+      action-xhr="http://iframe.localhost:8000/form/redirect-to/post"
+      target="_blank">
```

Comment:
In this form ( given `target="_blank" ) I would have totally expected the redirect to open a new window but it won't which is very confusing. Reading the documentation, looks like target is required but ignored in xhr cases. 

1) Why is target ignored in xhr cases? Is delayed window.open() to handle _blank blocked by popup blockers?
2) Why do we not support _self?
3) At minimum we should do something about the code above. target=_blank but redirect happening on top does not make sense. Can we change validation rule so either (target | action-xhr) is required?